office-365-management-api/office-365-management-activity-api-schema.md

```diff
@@ -823,6 +823,7 @@ DLP (Data Loss Prevention) events will always have UserKey="DlpAgent" in the com
 
 |**Parameters**|**Type**|**Mandatory?**|**Description**|
 |:-----|:-----|:-----|:-----|
+|ConditionMatchedInNewScheme|Edm.Boolean|Yes|Indicates whether the classification results in new scheme (high, medium and low) result in DLP rule match.|

```

Comment:
Suggestion: Description can be `Indicates whether the new classification scheme (high, medium and low) results in DLP rule match or not.`